#!/usr/bin/env python
PACKAGE = "carote"

from dynamic_reconfigure.parameter_generator_catkin import *

gen=ParameterGenerator()

gen.add("enable",int_t,0,"tweak activation flag",0,0,1)
gen.add("host",str_t,0,"subscriber host","127.0.0.1")
gen.add("port",int_t,0,"tweak port",2017,2017,2017)
gen.add("rate",double_t,0,"timer rate (in hertz)",50,20,100)
#gen.add("fifo",str_t,0,"FIFO path","/tmp/carote-system-tweak.fifo")
gen.add("input_state",str_t,0,"input topic (full path)","/carote/target/state")
gen.add("output_state",str_t,0,"output topic (full path)","/carote/target/state")

exit(gen.generate(PACKAGE, "parameters", "Tweak"))
